### PR TITLE
Feature/crossref

### PR DESCRIPTION
**Pull request by jannistsiroyannis**

### CODE REVIEW

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -5,18 +5,37 @@
...
     </servlet>
 
     <servlet-mapping>
         <servlet-name>UnpaywallApi</servlet-name>
         <url-pattern>/v2/*</url-pattern>
     </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>CrossrefApi</servlet-name>
+        <url-pattern>/works/*</url-pattern>
```
Comment (niklasl) [inline src/main/webapp/WEB-INF/web.xml:37]: *Maybe* more clear as e.g. `/unpaywall/v2/*` and `/crossref/works/*`? (Just a hunch, I know to little about the domain differences.)

**Review by niklasl — APPROVED** [review]
LGTM! (Might be nice to consider the comment/question about the app URI space.)

File: src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (jannistsiroyannis) [reply src/main/webapp/WEB-INF/web.xml:37]: The idea is/was to use exactly the same path that the original services use, so you only have the change the domain part of the url when swithching from real source to mirror or back.
There's also the slight annoyance that if we change the paths, a corresponding fix/release for swepub would also be needed to correct the path there :(

Comment (niklasl) [reply src/main/webapp/WEB-INF/web.xml:37]: Kinda figured that. Just wanted a more cohesive proposal than "perhaps call the new one /crossref/works/*". Of course, an alias for the existing `/v2/*` could work, but for was is basically an internal service, there is no need to clarify/disambiguate here now. If we add 2 or three more, perhaps.

Comment (jannistsiroyannis) [reply src/main/webapp/WEB-INF/web.xml:37]: I agree!

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ When you've obtained a download link do the following:
 1. Start the service up again
 
 
-The first time the service starts (with a new datadump) it will spend some time building an index of the dump. This will only happen once. This process may take an hour or so.
+The first time the service starts (with a new datadump) it will spend some time building an index of the dump. This will only happens once. This process may take an hour or so.
```
Comment (niklasl) [inline README.md]: Half-way between changing from "This will only happen once." to "This only happens once."? :wink:

Comment (jannistsiroyannis) [reply README.md]: Urghh, the fail is strong with this one. Ill change it back. Haha

**Comment by olovy** [comment]
Nice!

**Pull request merged**